telosnetwork/open-block-explorer
# 486 error reading actions on block page

# Fixes #486 

## Description
Handles blocks with zero tx actions

## Test scenarios
- visit `block/181847691` and see block data loaded and no errors in dev console
- visit `block/260845898` (or any block with tx actions) and see action count and transactions listed normally

## Checklist:
